kendzi3d-plugin/src/main/java/kendzi/josm/kendzi3d/Kendzi3DPlugin.java

```diff
@@ -165,6 +170,10 @@ public void actionPerformed(ActionEvent e) {
 
         view3dJMenu.addSeparator();
 
+        /* unfinished + may be better suited to fit in light configuration dialog */
+        DoubleSidedLightingToggleAction doubleSidedLightingToggleAction = injector.getInstance(DoubleSidedLightingToggleAction.class);
+        registerCheckBoxAction(doubleSidedLightingToggleAction, view3dJMenu);
+
```

Comment:
As it is an unfinished experimental option (and better be turned on/off on a per model basis),
commenting it may be better.  This way someone developing can easily re-enable it, to e.g.
check if this may be an idea worth following in an attempt to fix display bugs or other glitches.

I.e. it is rather an option to develop the plugin itself, than a productively usable one.  Or maybe
you make an expert option.  So that JOSM users only see it, when the "expert" option is used.